sources/src/resources/init-scripts/gradle-actions.inject-develocity.init.gradle

```diff
@@ -90,75 +97,115 @@ if (gradleInjectionEnabled != "true") {
-                            buildScan.server = geUrl
-                            buildScan.allowUntrustedServer = geAllowUntrustedServer
+                pluginManager.withPlugin(BUILD_SCAN_PLUGIN_ID) {
+                    afterEvaluate {
+                        if (develocityUrl && develocityEnforceUrl) {
+                            logger.lifecycle("Enforcing Develocity: $develocityUrl, allowUntrustedServer: $develocityAllowUntrustedServer for Build Scan plugin")
+                            buildScan.server = develocityUrl
+                            buildScan.allowUntrustedServer = develocityAllowUntrustedServer
+                        }
+
+                        if (buildScanTermsOfUseUrl && buildScanTermsOfUseAgree) {
+                            buildScan.termsOfServiceUrl = buildScanTermsOfUseUrl
+                            buildScan.termsOfServiceAgree = buildScanTermsOfUseAgree
                         }
-                        if (isAtLeast(gePluginVersion, '2.1') && atLeastGradle5) {
-                            if (isAtLeast(gePluginVersion, '3.7')) {
-                                buildScan.capture.taskInputFiles = geCaptureFileFingerprints
+
+                        if (isAtLeast(develocityPluginVersion, '2.1') && atLeastGradle5) {
+                            if (isAtLeast(develocityPluginVersion, '3.7')) {
+                                buildScan.capture.taskInputFiles = develocityCaptureFileFingerprints
                             } else {
-                                buildScan.captureTaskInputFiles = geCaptureFileFingerprints
+                                buildScan.captureTaskInputFiles = develocityCaptureFileFingerprints
                             }
                         }
                     }
                 }
+                pluginManager.withPlugin(DEVELOCITY_PLUGIN_ID) {
+                    afterEvaluate {
+                        if (develocityUrl && develocityEnforceUrl) {
+                            logger.lifecycle("Enforcing Develocity: $develocityUrl, allowUntrustedServer: $develocityAllowUntrustedServer for Build Scan plugin")
+                            develocity.server = develocityUrl
+                            develocity.allowUntrustedServer = develocityAllowUntrustedServer
+                        }
+
+                        if (buildScanTermsOfUseUrl && buildScanTermsOfUseAgree) {
+                            develocity.buildScan.termsOfUseUrl = buildScanTermsOfUseUrl
+                            develocity.buildScan.termsOfUseAgree = buildScanTermsOfUseAgree
+                        }
 
-                if (buildScanTermsOfServiceUrl && buildScanTermsOfServiceAgree) {
-                    buildScan.termsOfServiceUrl = buildScanTermsOfServiceUrl
-                    buildScan.termsOfServiceAgree = buildScanTermsOfServiceAgree
+                        develocity.buildScan.capture.fileFingerprints.set(develocityCaptureFileFingerprints)
```

Comment:
Can you set the new property after the project has been evaluated?

Reply:
No, that was wrong. Restored in https://github.com/gradle/actions/pull/62/commits/d19d801ed6180a8918df48c156960ff3c8519a4a